snippets/datasets-rubric.md

```diff
@@ -1,2 +1,3 @@
-If you require access to any of the above, the host organisations providing the data may also need to approve access to these datasets; there will also be associated acknowledgement publication policies that will need to be adhered to.
-OpenSAFELY will contact the applicant to discuss these (and the associated policies will also be made available on our documentation in due course).
+If you require access to any of the below datasets, the host organisations providing the data may also need to approve access to these datasets; there will also be associated [acknowledgement and data sharing / publication policies])https://policies-for-researchers.opensafely.pages.dev/policies-for-researchers/#acknowledgment-and-data-sharing--publication-policy) that will need to be adhered to. 
```

Comment:
You have a closing parenthesis instead of an opening parenthesis at the start of the data sharing link.